pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto

```diff
@@ -14,8 +14,16 @@
 
 syntax = "proto3";
 
+// $title: Operator Installation
+// $description: Configuration for Istio control plane installation through the Operator.
+// $location: https://istio.io/docs/reference/config/istio.operator.v1alpha12.pb.html
+
 package v1alpha2;
 
+// TODO: for some reason the below fails proto compilation in this repo.
```

Comment:
Can you link to an issue for this TODO unless you're planning to fix this in a follow-up?

Reply:
https://github.com/istio/istio/issues/16899

Reply:
Thanks!